models/artcles-models.js

```diff
@@ -9,4 +9,28 @@ exports.fetchArticlesById = (article_id) => {
       }
       return result.rows[0];
     })
+};
+
+exports.fetchArticles = (sort_by = 'created_at', order = 'desc') => {
```

Comment:
These options aren't yet necessary you can just have them hardcoded into you sql query

Reply:
As we currently don't require any flexibility